great_expectations/rule_based_profiler/data_assistant/data_assistant.py

```diff
@@ -93,8 +93,9 @@ def __init__(
         """
         DataAssistant subclasses guide "RuleBasedProfiler" to contain Rule configurations to embody profiling behaviors,
         corresponding to indended exploration and validation goals.  Then executing "RuleBasedProfiler.run()" yields
-        "RuleBasedProfilerResult" object, containing metrics by "Domain", list of "ExpectationConfiguration" objects,
-        and overall "ExpectationSuite" object, immediately available for validating underlying data "Batch" objects.
+        "RuleBasedProfilerResult" object, containing "fully_qualified_parameter_names_by_domain",
+        "parameter_values_for_fully_qualified_parameter_names_by_domain", "expectation_configurations", and "citation",
+        immediately available for composing "ExpectationSuite" and validating underlying data "Batch" objects.
```

Comment:
Do we have a mechanism for turning this result into an `ExpectationSuite` today?

Reply:
@NathanFarmer Yes -- illustrated in `tests/test_fixtures/rule_based_profiler/example_notebooks/DataAssistants_Instantiation_And_Running.ipynb`.  Thanks!